README.md

```diff
@@ -12,32 +15,39 @@ Clone or download the code from this GitHub. The stable version is the acceptanc
 
 ### Requirements
 
-Android Studio Arctic Fox | 2020.3.1 Patch 4 (latest tested version)   
-Android smartphone with at least SKD version 21 (Android 5.0 Lollipop) or emulator with equivalent specs.
+Up to date Android Studio
+Android smartphone with at least SDK version 21 (Android 5.0 Lollipop) or emulator with equivalent specs.
 
 ### Installation
 
-Open Android Studio and open a new project.  
-Go to 'Build > Generate Signed Bundle/APK > Build APK  
-Android Studio will download an APK to your PC, click the popup to locate the file.  
-The file is named something with 'APP_DEBUG', rename to whatever you want and sideload/install it on your phone.
-
-**Alternative**  
-Set up a Virtual Device using Android Studio. Klik the 'run' button to load the app on your virtual device.
+Open the source code in Android Studio en build the project.
+After building, run the application in an emulator or your actual Android phone.
 
 ## Credits
 
 * **Mitch Kessels** - _Creator_ - [MiKessels](https://github.com/MiKessels)
+* **Joey Vonck** - _Frontend_ - [joeyvonck](https://github.com/joeyvonck)
+* **Brit Prop** - _Frontend_ - [Prupke](https://github.com/Prupke)
+* **Cyril de Wit** - _Backend_ - [cyrildewit](https://github.com/cyrildewit)
+* **Jasper Stolwijk** - _Backend_ - [Hoopless](https://github.com/Hoopless)
+
+### Former team members
 * **Abas Sharif** - _Creator_ - [abassharif](https://github.com/abassharif)
-* **Joey Vonck** - _Creator_ - [joeyvonck](https://github.com/joeyvonck)
-* **Cyril de Wit** - _Creator_ - [cyrildewit](https://github.com/cyrildewit)
 
 
+## License
+
+This project is licensed under the MIT License - ~~see the LICENSE.md file for details.~~
+
+## Docs
+
+~~see the DOCS for details~~
+
 ## Changelog
```

Comment:
Are you familiar with keepachangelog.com? I also think that the changelog belongs in it's own file. See https://github.com/cyrildewit/eloquent-viewable/blob/master/CHANGELOG.md for example or see what other open source projects do.

Reply:
Yep, you told me earlier. Didn't start using changelog yet. I will delete the changelog for now and create a different file for the changelog, like ' the docs'